xml/deployment_installation.xml

```diff
@@ -825,14 +825,350 @@
+      You will need a &productname; machine image and &ostack; Heat templates.
+      Once you have created a stack, you will continue with the &productname; setup.
+     </para>
+     <note>
+      <title>&productname; machine image for &soc;</title>
+      <para>
+       Download the latest &productname; for &ostack; image from 
+       <link xlink:href="https://download.suse.com">https://download.suse.com</link>
+       (for example, 
+       <filename>SUSE-CaaS-Platform-3.0-OpenStack-Cloud.x86_64-1.0.0-GM.qcow2</filename>).
+      </para>
+     </note>
+      <note>
+       <title>
+        &ostack; Heat Templates Repository
+       </title>
+       <para>
+        &productname; Heat templates are available from <link xlink:href="https://github.com/SUSE/caasp-openstack-heat-templates">GitHub</link>.
+       </para>
+     </note>
+     
+     <sect3>
+      <title>Using the Horizon Dashboard</title>
+      <procedure>
+
+      <step>
+       <para>
+        Go to <guimenu>Project &rarr; Compute &rarr; Images</guimenu> and click on 
+        <guimenu>Create Image</guimenu>.
+       </para>
+       <para>
+        Give your image a name (for example: <literal>CaaSP-3</literal>); you will
+        need to use this later to find the image.
+       </para>
+      </step>
+       <step>
+        <para>
+         Go to <guimenu>Project &rarr; Orchestration</guimenu> and click on 
+         <guimenu>Stacks</guimenu>.
+        </para>
+        <informalfigure>
+         <mediaobject>
+          <imageobject role="fo">
+           <imagedata fileref="horizon_stacks.png" width="100%"/>
+          </imageobject>
+          <imageobject role="html">
+           <imagedata fileref="horizon_stacks.png" width="100%"/>
+          </imageobject>
+         </mediaobject>
+        </informalfigure>
+       </step>
+       
+       <step>
+        <para>
+         Click on <guimenu>Launch Stack</guimenu>.
+        </para>
+       </step>
+       
+       <step>
+        <para>
+         You need to provide the stack templates at this stage.
+         Either upload the files, provide the URL to the raw files directly 
+         (only applies to stack template), or copy and paste the contents into the 
+         <literal>Direct Input</literal> fields.
+        </para>
+        <warning>
+         <title>Replace the default <replaceable>root_password</replaceable></title>
+          <para>
+           Do not use the <filename>caasp-environment.yaml</filename> directly
+           from the GitHub repository.
+          </para>
+           <para>
+            You must make sure to replace the value for 
+            <literal>root_password</literal> with a secure password.
+            This will become the password for the &rootuser; account on all 
+            nodes in the stack.
+           </para>
+         <screen>
+---
+parameters:
+root_password: <replaceable>linux</replaceable>
+admin_flavor: m1.large
+&hellip;
+         </screen>
```

Comment:
Personally I think we don't need the screen here at all.

But if you want it, I suggest using a title for the file name. As screens cannot have titles, you can wrap everything in &lt;example&gt;:
```
<example>
 <title>Example of a caasp-environment.yaml file</title>
 <screen>
 ...
 </screen>
</example>
```

Downside: Examples are listed in the appendix and as this warning box occurs twice, there will be two example caasp-environment.yaml files.